tools/testing/target_determination/heuristics/edited_by_pr.py

```diff
@@ -25,6 +25,10 @@ def get_test_priorities(self, tests: List[str]) -> TestPrioritizations:
 
         return test_rankings
 
+    def get_test_ratings(self, tests: List[str]) -> Dict[str, float]:
+        critical_tests = _get_modified_tests()
```

Comment:
These tests are sorted in `get_test_priorities`. Should it be sorted here as well?

Reply:
I don't think it needs to be since the output is a dictionary.  Presumably whatever calls get_test_ratings has to aggregate the dictionaries and then do it's own sorting based on the aggregation.